<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoutubeDl</class>
 <widget class="QMainWindow" name="YoutubeDl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to Youtube-dl Utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="url_and_paths_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>249</height>
     </rect>
    </property>
    <property name="title">
     <string>Youtube-dl Actions</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="url_label">
           <property name="text">
            <string>Please enter url</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="path_label">
           <property name="text">
            <string>Please enter path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="url_lineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="dir_lineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="explore_directory_radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="explore_directory_label">
         <property name="text">
          <string>Select a Directory                                      </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="clear_fields_radioButton">
       <property name="text">
        <string>Clear fields</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="print_url_title_radioButton">
         <property name="text">
          <string>Print url title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="print_url_format_radioButton">
         <property name="text">
          <string>Print url format</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="confirm_button">
       <property name="text">
        <string>CONFIRM TO PROCEED</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="provided_options_groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>401</width>
      <height>164</height>
     </rect>
    </property>
    <property name="title">
     <string>Provided Options</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="audio_label">
       <property name="text">
        <string>audio_label</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="audio_radioButton">
       <property name="text">
        <string>audio</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="download_button">
         <property name="text">
          <string>DOWNLOAD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="information_button">
         <property name="text">
          <string>INFORMATION</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="video_label">
       <property name="text">
        <string>video_label</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="video_radioButton">
       <property name="text">
        <string>video</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="download_status_label">
       <property name="text">
        <string>DOWNLOAD OUTCOME</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="download_status_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="ffmpge_actions_groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Audio and Video Conversion Actions</string>
    </property>
    <widget class="QPushButton" name="convert_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CONVERT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_ffmpeg_fields_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear ffmpeg Fields</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="chosen_file_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="sampling_rate_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="bitrate_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="channels_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="format_lineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QRadioButton" name="old_filename_radioButton">
        <property name="text">
         <string>Select file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sampling_rate_label">
        <property name="text">
         <string>Audio Sampling Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bitrate_label">
        <property name="text">
         <string>Audio bitrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sampling_rate_label_2">
        <property name="text">
         <string>Audio Channels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="format_label">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="open_folder_action"/>
    <addaction name="open_file_action"/>
    <addaction name="information_action"/>
    <addaction name="exit_application_action"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about_app_action"/>
    <addaction name="about_author_action"/>
    <addaction name="actionAbout_Youtube_dl_Utility"/>
   </widget>
   <widget class="QMenu" name="menuVersioning">
    <property name="title">
     <string>youtube-dl Actions</string>
    </property>
    <addaction name="actionyoutube_dl_command_version"/>
    <addaction name="actionInstall_youtube_dl_command"/>
    <addaction name="actionUpdate_youtube_dl_command"/>
    <addaction name="actionRemove_youtube_dl_command"/>
   </widget>
   <widget class="QMenu" name="menuffmpeg_Actions">
    <property name="title">
     <string>ffmpeg Actions</string>
    </property>
    <addaction name="actionffmpeg_version"/>
    <addaction name="actionInstall_ffmpeg_command"/>
    <addaction name="actionRemove_ffmpeg_command"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
   <addaction name="menuVersioning"/>
   <addaction name="menuffmpeg_Actions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_folder_action"/>
   <addaction name="open_file_action"/>
   <addaction name="information_action"/>
   <addaction name="separator"/>
   <addaction name="exit_application_action"/>
  </widget>
  <action name="open_folder_action">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/open_folder.png</normaloff>:/open_folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="information_action">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/help.png</normaloff>:/help.png</iconset>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="exit_application_action">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit Application</string>
   </property>
  </action>
  <action name="about_app_action">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/about.png</normaloff>:/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="about_author_action">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/about_author.png</normaloff>:/about_author.png</iconset>
   </property>
   <property name="text">
    <string>About Author</string>
   </property>
  </action>
  <action name="actionAbout_Youtube_dl_Utility">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/ytb.png</normaloff>:/ytb.png</iconset>
   </property>
   <property name="text">
    <string>About Youtube-dl Utility</string>
   </property>
  </action>
  <action name="actionyoutube_dl_command_version">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/version.png</normaloff>:/version.png</iconset>
   </property>
   <property name="text">
    <string>youtube-dl command version</string>
   </property>
  </action>
  <action name="actionInstall_youtube_dl_command">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/install.png</normaloff>:/install.png</iconset>
   </property>
   <property name="text">
    <string>Install youtube-dl command</string>
   </property>
  </action>
  <action name="actionUpdate_youtube_dl_command">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/update.png</normaloff>:/update.png</iconset>
   </property>
   <property name="text">
    <string>Update youtube-dl command</string>
   </property>
  </action>
  <action name="actionRemove_youtube_dl_command">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove youtube-dl command</string>
   </property>
  </action>
  <action name="actionffmpeg_version">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/version.png</normaloff>:/version.png</iconset>
   </property>
   <property name="text">
    <string>ffmpeg command version</string>
   </property>
  </action>
  <action name="actionInstall_ffmpeg_command">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/install.png</normaloff>:/install.png</iconset>
   </property>
   <property name="text">
    <string>Install ffmpeg command</string>
   </property>
  </action>
  <action name="actionRemove_ffmpeg_command">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove ffmpeg command</string>
   </property>
  </action>
  <action name="open_file_action">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/open_file.png</normaloff>:/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
